src/pattern-library/router.tsx

```diff
@@ -1,16 +1,12 @@
 import { useCallback, useEffect, useMemo, useState } from 'preact/hooks';
 
-/**
- * @typedef {import('./routes').PlaygroundRoute} PlaygroundRoute
- */
+import type { PlaygroundRoute } from './routes';
```

Comment:
The functionality of this module needs some work but I believe we may be about to settle on a general front-end routing approach so I'm holding off.